packages/core/src/shared/utilities/proxyUtil.ts

```diff
@@ -73,9 +70,6 @@ export class ProxyUtil {
         // Always enable experimental proxy support for better handling of both explicit and transparent proxies
         process.env.EXPERIMENTAL_HTTP_PROXY_SUPPORT = 'true'
 
-        // Load built-in bundle and system OS trust store
-        process.env.NODE_OPTIONS = '--use-system-ca'
```

Comment:
this line makes sense, but it is unclear to me how duplicate certificates in the trust list will cause an issue downstream in flare

Reply:
Since we lack instrumentation for this issue I would prefer we are less selective by pulling in all available certs